### ISSUE TITLE
Array Errors Structure

### ISSUE THREAD

**Issue by DownChapel** [ciscoheat/sveltekit-superforms#411]
I'm coming across a difference in the structure of array errors depending on the schema of the parent object and wondering if it's a bug. Apologies if this isn't the right place to ask this type of question.

I have an array schema that is used in multiple places.

E.g. a simple array schema with a min length: 

```javascript
const testArraySchema = z.array(z.string()).min(1);
```

This is used in two places, one where the parent object is an array, one where it isn't:

```javascript
const schema = z.object({
	values: z.array(
		z.object({
			testArray: testArraySchema
		})
	),
	defaultValue: z.object({
		testArray: testArraySchema
	})
});
```

However, the array min length error is in the errors structure in two different ways. One inside `_errors` and one without:

```json
{
  "values": {
    "0": {
      "testArray": { "_errors": ["Array must contain at least 1 element(s)"] }
    }
  },
  "defaultValue": { "testArray": ["Array must contain at least 1 element(s)"] }
}
```

In both cases `testArray` is an array so shouldn't the error structure be the same?

I've done a minimal reproduction of the issue below. 

```svelte
<script lang="ts">
	import { z } from 'zod';
	import { defaults, superForm } from 'sveltekit-superforms';
	import { zod } from 'sveltekit-superforms/adapters';

	const testArraySchema = z.array(z.string()).min(1);

	const schema = z.object({
		values: z.array(
			z.object({
				testArray: testArraySchema
			})
		),
		defaultValue: z.object({
			testArray: testArraySchema
		})
	});

	const defaultSchemaValue = {
		defaultValue: {
			testArray: []
		},
		values: [
			{
				testArray: []
			}
		]
	};
	const entireForm = superForm(defaults(defaultSchemaValue, zod(schema)), {
		SPA: true,
		dataType: 'json',
		validators: zod(schema),
		onUpdate({ form }) {
			console.log(form.valid);
		}
	});

	const { enhance, errors } = entireForm;
</script>

{JSON.stringify($errors)}

<form method="POST" use:enhance novalidate>
	<button type="submit">Submit</button>
</form>
```

If you submit the form, the errors structure is shown as above.

https://www.sveltelab.dev/ip7f8znnmkcd1u2

Thank you!

**Comment by ciscoheat** [comment]
Fixed with 2.12.6

**Comment by DownChapel** [comment]
@ciscoheat Thank you!
